aev/src/main/java/com/fingerprintjs/android/aev/ApiInteractor.kt

```diff
@@ -1,18 +1,20 @@
 package com.fingerprintjs.android.aev
 
 
+import com.fingerprintjs.android.aev.errors.ApiInteractorGetTokenError
 import com.fingerprintjs.android.aev.logger.Logger
 import com.fingerprintjs.android.aev.requests.InitVerifyRequest
-import com.fingerprintjs.android.aev.requests.InitVerifyResult
 import com.fingerprintjs.android.aev.requests.InitVerifyResponse
 import com.fingerprintjs.android.aev.signals.SignalProvider
 import com.fingerprintjs.android.aev.transport.HttpClient
+import com.cloned.github.michaelbull.result.Result
+import com.cloned.github.michaelbull.result.flatMap
 
 
 internal interface ApiInteractor {
-    fun getToken(
+    fun getRequestId(
         signalProvider: SignalProvider
-    ): InitVerifyResponse
+    ): Result<InitVerifyResponse, ApiInteractorGetTokenError>
```

Comment:
`Token` – was the first naming for `requestId`. If it's not difficult, let's complete changing naming